scss/utilities/_colours.scss

```diff
@@ -22,6 +22,7 @@ $cod-gray:	    #111111;
 $matisse: 		#206095; //links
 $blumine:       #1A4C76; //hover links
 $salem:			#0F8243; //active links
+$salem-success: #edf4f0; //success bg colour
```

Comment:
These should be the names of the colour rather than containing where they are used.
This colour `edf4f0` is known as `Lily White`